lib/ansible/module_utils/basic.py

```diff
@@ -74,7 +74,7 @@
 
 HAVE_SELINUX = False
 try:
-    import selinux
+    import ansible.module_utils.compat.selinux as selinux
```

Comment:
Do we still need this try-except/HAVE_SELINUX stuff? This import should never fail now.

Reply:
It should still fail;  in `ansible.module_utils.compat.selinux` this PR raises `ImportError` if `libselinux.so.1` cannot be found.

Reply:
Ah, see that now. Thx.

Reply:
(see below)